pages/_app.js

```diff
@@ -32,15 +33,24 @@ export default function App({ Component, pageProps }) {
 
   return (
     <>
-      <GlobalStyle />
-      <Component
-        {...pageProps}
-        projects={projects}
-        onAddProject={handleAddProject}
-        onDeleteProject={handleDeleteProject}
-        onToggleFavorite={handleToggleFavorite}
-        onEditProject={handleEditProject}
-      />
+      <div className="App" id="outer-container">
```

Comment:
You don't seem to be using the class App anyway, but we don't work with classes during the Capstone. If you need to style the overall Layout of the App, please use the `<Layout>` component from next.js